src/js/index.js

```diff
@@ -52,8 +52,10 @@ search().then((index) => {
   console.error(err);
 });
 
-window.ga = window.ga || function ga() {
-  (ga.q = ga.q || []).push(arguments);
+window.ga = window.ga || function ga(...args) {
+  (ga.q = ga.q || []).push(args);
+  // TODO consult if that's equal to:
```

Comment:
It is, but I'm not sure if you want to make `ga.q` to be an array of items or an array whose first item is an array of items. Given `ga` called as `ga(1,2,3)`. What do you want for `ga.q` to be?
  * `[1,2,3]`
  * Currently, you are doing this `[ [1,2,3] ]`

If you want the other, then it is: `(ga.q = ga.q || []).push(...args)`